apps/arweave/src/ar.erl

```diff
@@ -628,6 +627,10 @@ parse_cli_args(["pool_server_address", Host | Rest], C) ->
 	parse_cli_args(Rest, C#config{ pool_server_address = list_to_binary(Host) });
 parse_cli_args(["pool_worker_name", Host | Rest], C) ->
 	parse_cli_args(Rest, C#config{ pool_worker_name = list_to_binary(Host) });
+parse_cli_args(["rocksdb_flush_interval", Seconds | Rest], C) ->
+	parse_cli_args(Rest, C#config{ rocksdb_flush_interval_s = list_to_integer(Seconds) });
+parse_cli_args(["rocksdb_wal_sync_interval", Seconds | Rest], C) ->
```

Comment:
It would be good to also add these flags to ar_config.erl. We *try* to keep CLI and config.json at parity - the main exceptions are:
1. complex JSON-based config that isn't practical on the CLI (e.g. p3 config, semaphore config)
2. historical errors where a flag was accidentally only added to one or the other and hasn't been fixed

Reply:
Done.